tests/IBFE/explicit_ex4.cpp

```diff
@@ -366,11 +369,15 @@ main(int argc, char** argv)
         if (use_separate_fe_data_manager)
         {
             FEDataManager::WorkloadSpec spec;
+            IntVector<NDIM> min_ghost_width = 0;
+            std::shared_ptr<IBTK::SAMRAIDataCache> eulerian_data_cache(new IBTK::SAMRAIDataCache());
             other_manager = FEDataManager::getManager(fe_data_manager->getFEData(),
                                                       "cloned_fe_data_manager",
                                                       fe_data_manager->getDefaultInterpSpec(),
                                                       fe_data_manager->getDefaultSpreadSpec(),
-                                                      spec);
+                                                      spec,
+                                                      min_ghost_width,
+                                                      eulerian_data_cache);
```

Comment:
Is modifying this test necessary? Would it work in its current form?

Reply:
Nope, not necessary. I will revert.

Reply:
It is not a necessary change. I will revert.